src/proposals/SparkE2ETest.t.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import '../SparkTestBase.sol';
+
+contract SparkEthereum_E2ETest is SparkEthereumTestBase {
+
+    constructor() {
+        id = '20231129';
+    }
+
+    function setUp() public {
+        vm.createSelectFork(getChain('mainnet').rpcUrl, 18707048);
+        payload = 0x68a075249fA77173b8d1B92750c9920423997e2B;
+
+        loadPoolContext(poolAddressesProviderRegistry.getAddressesProvidersList()[0]);
+    }
+
+}
```

Comment:
File and contract do not match in name, plus I'd rename it to something more standard. Why not classic `SparkEthereum_20231129Test` or `SparkEthereum_20231129Test_E2E` in `src/proposal/20231129/SparkEthereum_20231129.t.sol`

Reply:
I think it's ok this will just be removed after we are satisfied the tests are good.

Reply:
moved to test dir